### PR TITLE
Quick debugging tool

### PR DESCRIPTION
**Pull request by deadlybutter**
This should make it easier for admins to peak at the Analytics/Signup storage stuff and also clear local storage when trying to test

screen shot 2017-04-13 at 11 27 34 am

### CODE REVIEW

**Comment by deadlybutter** [comment]
@weerd @DFurnes No rush on this - but might be helpful for debugging & testing today + tomorrow

**Comment by weerd** [comment]
Just a suggestion but maybe name component `Debug` or `Debugger`since you're not actually configuring anything in there, right?

File: resources/assets/components/Config/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import {
+  getArray as storageGetArray,
+  remove as storageRemove,
+  SIGNUP_STORAGE_KEY
+} from '../../helpers/storage';
+import { getSession } from '../../helpers/analytics';
+import './config.scss';
+
+const Config = (props) => {
+  if (props.user.role !== 'admin') return null;
```
Comment (weerd) [inline resources/assets/components/Config/index.js]: ye ole' brackets `{ ... }` :)

Comment (weerd) [inline resources/assets/components/Config/index.js]: even for one liners, when we eventually add some eslinting, it'll yell at us based on our DS eslint config.

File: resources/assets/components/Config/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  return (
+    <div className="config">
+      <Flex>
+        <FlexCell width="one-third">
+          <div>
+            <h1>analytics debug</h1>
```
Comment (weerd) [inline resources/assets/components/Config/index.js]: all lowercase for an `<h1>`. bold sir.

Comment (deadlybutter) [inline resources/assets/components/Config/index.js]: cc: NYTCopyDesk

**Comment by deadlybutter** [comment]
@weerd I think this reads better! Seems to still work

**Comment by deadlybutter** [comment]
ALSO PSA

This delete signup button *DOES NOT* update the store (Trying to timebox this..). So you have to refresh the page to see any updates. (And remember to delete your signup on Phoenix)

**Pull request merged**